src/theme/default/styled.d.ts

```diff
@@ -0,0 +1,8 @@
+import 'styled-components'
+import { defaultTheme } from '.'
+
+declare module 'styled-components' {
+  type Theme = typeof defaultTheme
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
Na projektu máme pravidlo nepoužívání prázdných komentářů. Exisutjí tedy 2 varianty: 

- Pobavit se o tom, zda pravidlo má smysl. 
- Vyřešit problém, který ESLint ukazuje 

Osobně jsem pro druhou variantu, protože můžeme už zadefinovat stávající rozhraní a v dalších PRs ho postupně rozšiřovat/měnit. Co si o tom myslíš?

Reply:
Bohužel, nemůžu napsat `export type DefaultTheme = Theme`, protože pokud chápu správně, TS si stěžuje, že symbol se stejným jménem již byl zadeklarovaný ve vnitřnostech styled-components - nemůže vyexportovat `type` a `interface` se stejným názvem. Další možnost `export interface DefaultTheme extends Theme` bez závorek není validní TS syntaxe. Komentáře rušící eslint pro jeden řádek mi přišel lepší, než přidat nějakou dummy propertu.